lgn-provers/src/provers/v1/groth16/task.rs

```diff
@@ -33,7 +33,7 @@ impl LgnProver for Groth16Prover {
             let reply_envelope = MessageReplyEnvelope::new(query_id, task_id, reply_type);
             Ok(reply_envelope)
         } else {
-            bail!("Unexpected task type: {:?}", envelope.inner());
```

Comment:
All error messages logging `envelope`, previously it was a mix of that and `envelope.inner()`.